### PR TITLE
Fixed Issue #1006

### PR DESCRIPTION
**Pull request by vintagecodes**

### CODE REVIEW

File: templates/emails/vendor-new-announcement.php [hunk 1]
```diff
@@ -20,6 +20,7 @@
 <?php $announcement_link = esc_url(wcmp_get_vendor_dashboard_endpoint_url(get_wcmp_vendor_settings('wcmp_vendor_announcements_endpoint', 'vendor', 'general', 'vendor-announcements'))); ?>
 
 <p><?php printf(esc_html__('This is to inform you that we recently updated the article %s :', 'dc-woocommerce-multi-vendor'), $post_title); ?></p>
+<p><?php printf(esc_html__('Vendor Name: %s','dc-woocommerce-multi-vendor'), $vendor->page_title); ?></p>
```
Comment (cool-raju) [inline templates/emails/vendor-new-announcement.php:23]: give space after ,

**Review by cool-raju — APPROVED** [review]

File: templates/emails/plain/vendor-new-announcement.php [hunk 2]
```diff
@@ -21,6 +21,8 @@
 
 echo sprintf(__('This is to inform you that we recently updated the article %s :','dc-woocommerce-multi-vendor'), $post_title). "\n";
 
+echo sprintf(_('Vendor Name:  %s','dc-woocommerce-multi-vendor'), $vendor->page_title)."\n";
```
Comment (cool-raju) [inline templates/emails/plain/vendor-new-announcement.php:24]: why _( ???
Update translate file an check this vendor name coming or not

**Review by cool-raju — APPROVED** [review]

File: templates/emails/plain/vendor-new-announcement.php (hunk 2 above)
Comment (vintagecodes) [reply templates/emails/plain/vendor-new-announcement.php:24]: Yes it is coming

Comment (vintagecodes) [reply templates/emails/plain/vendor-new-announcement.php]: Updated the _( to __(

**Pull request closed without merging**